src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,11 +2,11 @@
 
 Release History
 ===============
-
-* Remove erroneous print statement for `az webapp auth update`
+* webapp, functionapp: az webapp/functionapp deployment list-publishing-credentials, get the Kudu (scm) url and its credentials
 
 0.2.14
 ++++++
+* Remove erroneous print statement for `az webapp auth update`
```

Comment:
why you moved anot yet released change to be a released one?

Reply:
changed.